airbyte-cron/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+micronaut:
+  application:
+    name: airbyte-cron
+
+airbyte:
+  worker:
+    env: ${WORKER_ENVIRONMENT:DOCKER}
```

Comment:
Using the same structure as in https://github.com/airbytehq/airbyte/blob/c4fb6b7f9eb9c371902e8c4bec484c2427954cf4/airbyte-workers/src/main/resources/application.yml#L87-L88